gpkit/constraints/array.py

```diff
@@ -0,0 +1,17 @@
+"Implements ArrayConstraint"
+from .set import ConstraintSet
+from .single_equation import SingleEquationConstraint
+
+
+class ArrayConstraint(ConstraintSet, SingleEquationConstraint):
+    """A ConstraintSet for prettier array-constraint printing.
+
+    Because ArrayConstraint also inherits from ConstraintSet,
+    its substitutions is recursive and so left and right are
```

Comment:
` its substitutions is recursive` is a bit confusing because substitutions do not appear in this file.